angelsbioprocessing/prototypes/recipes/bio-processing-paper.lua

```diff
@@ -65,12 +66,13 @@ data:extend(
         {type = "fluid", name = "gas-carbon-dioxide", amount = 70}
       },
       always_show_products = true,
-      icons = angelsmods.functions.create_gas_recipe_icon(
+      icons = AF.create_gas_recipe_icon(
         {
           {"__angelspetrochem__/graphics/icons/molecules/carbon-dioxide.png", 72}
         },
-        "ccc"
+        "COO"
```

Comment:
The second and third 'C' make it dark red, rather than the vibrant red of oxygen... I would leave this in the icon...
![image](https://user-images.githubusercontent.com/26593477/140639091-1dc15ef6-a1cb-4c42-8ae3-047b225ec3ef.png)

Reply:
I was leaning on some of the same functions, and thought I may consolidate it a bit, but I can change that back... I think only the C's were changed

Reply:
Yes, only the C's

Reply:
I made a "middle" ground... adding the darker red tints The CO and CO2 icons now look different as a result